htdocs/langs/en_US/cashdesk.lang

```diff
@@ -145,3 +145,6 @@ ShowProductReference=Show reference of products
 UsePriceHT=Use price excl. taxes and not price incl. taxes
 TerminalName=Terminal %s
 TerminalNameDesc=Terminal name
+DefaultPOSThirdLabel=TakePOS sales
+DefaultPOSCatLabel=Point of sale
```

Comment:
I suggest to name this category like this
"Category for all products in Point Of Sale"

Reply:
Approved for me